src/redux/products/operation.ts

```diff
@@ -7,33 +7,48 @@ import {
 } from '../../types'
 import axiosInstance from '../../axios/settings'
 
-type FetchItemOperationType = 'loadMore' | 'fetch'
+type FetchItemOperationType =
+  | 'loadMore'
+  | 'fetch'
+  | FetchAllCategoriesOperationType
+  | null
 
 type FetchItemsResponse = {
   data: ProductResponse[]
   operationType: FetchItemOperationType
+  totalCount: number
 }
 
 type FetchItemsParams = {
-  params: number
+  offset: number
+  sortValue: string
   operationType: FetchItemOperationType
+  category: string | false | undefined
```

Comment:
try to use one basic type, cause it looks messy, and hard to understand what should i pass and what value it will be